app/src/main/java/com/glowstudio/android/blindsjn/feature/main/view/MainScreen.kt

```
@@ -0,0 +1,160 @@
+                            onCancel = { navController.navigateUp() },
+                            onSave = { schedule ->
+                                // TODO: 일정 저장 로직 구현
+                                navController.navigateUp()
+                            }
+                        )
+                    }
+                    composable("news_detail/{articleJson}") { backStackEntry ->
+                        val articleJson = backStackEntry.arguments?.getString("articleJson")
+                        val article = try {
+                            Gson().fromJson(URLDecoder.decode(articleJson, "UTF-8"), Article::class.java)
+                        } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)